lib/GraphQLAdapter.js

```diff
@@ -1,3 +1,5 @@
+const cds = require('@sap/cds')
+require('./api').registerCompileTargets(cds.compile.to)
```

Comment:
Necessary?

Reply:
Not necessary, since plugin should always be loaded as long as it exists as a dependency